src/Perspex.Controls/Canvas.cs

```diff
@@ -0,0 +1,131 @@
+﻿// Copyright (c) The Perspex Project. All rights reserved.
```

Comment:
Wait... What is Canvas.cs doing in a commit called "Panel Background Brush"? Wasn't Canvas.cs already submitted?